awx/ui/test/e2e/tests/test-websockets.js

```diff
@@ -56,12 +57,12 @@ module.exports = {
         client
             .useCss()
             .navigateTo(`${AWX_E2E_URL}/#/home`);
-        getJob('test-websockets', 'debug.yml', 'test-websockets-successful');
+        getJob('test-websockets', 'debug.yml', 'test-websockets-successful', false);
```

Comment:
I'm on mobile, so take this review at face value. Do we want to leave this debug statement on or is this used to parse information in tests?

Reply:
debug.yml is a very fast, default playbook i'm using from `test-playbooks` instead of `hello_world.yml`